From 9c46ac2e38736c8eef202f0b31feca9e56e7e712 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:03 -0400
Subject: [redhat] kabi: Adding symbol scsi_is_sdev_device (drivers/scsi/scsi_sysfs.c)

Message-id: <62e0c82f99b23996689da22a58aa33bff4db6dd9.1410510773.git.poros@redhat.com>
Patchwork-id: 96005
O-Subject: [PATCH RHEL7.1 183/233] kabi: Adding symbol scsi_is_sdev_device (drivers/scsi/scsi_sysfs.c)
Bugzilla: 1062361
RH-Acked-by: Rob Evers <revers@redhat.com>

Bugzilla: 1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361

    Description:
    ============
    PowerPath uses this function to verify if a device
    is a scsi device (i.e. of type scsi_dev_type).

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/scsi_is_sdev_device
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/scsi_is_sdev_device
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/scsi_is_sdev_device

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/scsi_is_sdev_device b/redhat/kabi/kabi-module/kabi_ppc64/scsi_is_sdev_device
new file mode 100644
index 0000000..73c963f
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/scsi_is_sdev_device
@@ -0,0 +1,2 @@
+#1-
+0x86663310 scsi_is_sdev_device vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/scsi_is_sdev_device b/redhat/kabi/kabi-module/kabi_s390x/scsi_is_sdev_device
new file mode 100644
index 0000000..3a15268
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/scsi_is_sdev_device
@@ -0,0 +1,2 @@
+#1-
+0x07986ae0 scsi_is_sdev_device vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/scsi_is_sdev_device b/redhat/kabi/kabi-module/kabi_x86_64/scsi_is_sdev_device
new file mode 100644
index 0000000..1ea4b0c
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/scsi_is_sdev_device
@@ -0,0 +1,2 @@
+#1-
+0xf790d854 scsi_is_sdev_device vmlinux EXPORT_SYMBOL
-- 
1.7.1